templates/jdk21/resources/usr/local/bin/set_java_version.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+function create_symlink {
```

Comment:
@Wykenz, can't we reuse this same file presented in the folder `templates/jdk11/resources`

Reply:
Hi @natocesarrego, I tried to do this but as I can see the building process has a pattern on how the files can be made and used.
Every image have a build script, for zulu21 I made [build_jdk21_image.sh](https://github.com/Wykenz/liferay-docker/blob/LPD-23797-generate-docker-images-using-java-zulu-21/build_jdk21_image.sh ) based on the jdk11's script. There is this function [make_temp_directory](https://github.com/Wykenz/liferay-docker/blob/LPD-23797-generate-docker-images-using-java-zulu-21/build_jdk21_image.sh#L48) which is from [_common.sh](https://github.com/Wykenz/liferay-docker/blob/LPD-23797-generate-docker-images-using-java-zulu-21/_common.sh#L184) that creates a tempdir with all the contents of the image, and this function is used by every build_image.sh files. The other images have its resources under their own directory so I feel like separating them would go against the standard.

Reply:
Hi @Wykenz, you can do some little changes, as highlighted in red in the image below, to reuse the code, please see:

![image](https://github.com/natocesarrego/liferay-docker/assets/6729287/6bdc2c82-9635-42a8-8375-d6768c1268fd)

Can you try to move on like this?